apps/src/javalab/JavabuilderConnection.js

```diff
@@ -349,15 +350,19 @@ export default class JavabuilderConnection {
 
   onUnsupportedNeighborhoodMessage() {
     if (!this.seenUnsupportedNeighborhoodMessage) {
-      this.onOutputMessage(javalabMsg.unsupportedNeighborhoodMessage());
+      this.onOutputMessage(
+        '[EXCEPTION] ' + getUnsupportedMiniAppMessage(CsaViewMode.NEIGHBORHOOD)
```

Comment:
should `[EXCEPTION]` be part of the i18n string so it's translatable?

Reply:
oh good point, we have a constant `EXCEPTION_PREFIX` in our constants file but it's not translatable. We should probably move that into an i18n string. We could just set the constant to equal a new i18n string so we don't have to change it everywhere I think

Reply:
Yeah good call, I'll add that

Reply:
yeah, although it gets tricky when you start concatenating i18n strings because in RTL the order would be backwards (it would be "error message [EXCEPTION]")

Reply:
Looks like the `EXCEPTION_PREFIX` is only actually used here, so I think for now I'll create a new string with the prefix and the message (`"[EXCEPTION] {details}"`) and use that in place of the constant. That way at least the translation will be consistent even if "EXCEPTION" doesn't end up getting translated.